pb/nodemessage.proto

```diff
@@ -1,268 +1,308 @@
-syntax = "proto3";
```

Comment:
Something is wrong here, this file is recognized as completely rewritten. Could be your IDE configuration